conf/modules.config

```diff
@@ -274,6 +274,10 @@ process {
     }
 
     withName: PORECHOP_ABI {
+        ext.args = [
+            params.longread_qc_predictadapters ? "--ab_initio --discard_database" :
+            params.longread_qc_adapterlist ? "--custom_adapters ${params.longread_qc_adapterlist} --discard_database": ""
```

Comment:
I just realised this won't work - the adapterlist is a file in this case right? This has to come via an input channel to the module.